.github/workflows/clear-all-data-dev.yaml

```diff
@@ -0,0 +1,49 @@
+name: Clear all data - DEV
+on:
+  workflow_dispatch:
+env:
+  ENV_NAME: dev
+
+jobs:
+  clear-all-data:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    environment:
+      name: dev
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Cache yarn
+        uses: actions/cache@v3
+        with:
+          path: ./.yarn/cache
+          key: pcc-backend
+          restore-keys: pcc-backend
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-skip-session-tagging: true
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY_ID }}
+          aws-region: ca-central-1
+          role-to-assume: ${{ secrets.AWS_SA_ROLE_ARN }}
+          role-duration-seconds: 900
+          role-session-name: dev-data-clean
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Setup ENV file
+        run: cp ./.config/.env.example ./.env
+
+      - name: Clear S3 bucket
+        run: make aws-empty-s3-bucket-dev
+
+      - name: Open DB tunnel
+        run: make open-db-tunnel
+
+      - name: Clear DB
+        run: make clear-database
```

Comment:
This looks good. I think we might be missing a couple of things from the make clear-database (new items that should be cleared...)

`delete from program_daily_upload`

and

`delete from file_uploaded`

I am happy to create a separate PR for these (I should have added them in previously, it is not part of this for this PR but just FYI)

Reply:
Oh shoot, I should have reviewed that more closely. I can add in another commit!

Reply:
Added